pkg/controller/resourcequota/resource_quota_monitor.go

```diff
@@ -268,7 +268,7 @@ func (qm *QuotaMonitor) StartMonitors() {
 	for _, monitor := range monitors {
 		if monitor.stopCh == nil {
 			monitor.stopCh = make(chan struct{})
-			qm.informerFactory.Start(qm.stopCh)
+			qm.informerFactory.StartWithStopOptions(qm.stopCh)
 			go monitor.Run()
```

Comment:
GC and RQ are using monitors similarly, not sure why you're treating them differently for GC and RQ

Reply:
I'm confused, can you explain what you mean by "treating them differently" for GC and RQ?